NativeFileDialogs.CppSharpGenerator/NativeFileDialogsLibrary.cs

```diff
@@ -24,7 +24,7 @@ public void Setup(Driver driver)
     {
         Directory.CreateDirectory(include_directory);
 
-        string fileText = File.ReadAllText(@"../nativefiledialog-extended/src/include/nfd.h")
+        string fileText = File.ReadAllText(@"../../../../nativefiledialog-extended/src/include/nfd.h")
```

Comment:
What? That makes no sense. This file is exactly one level deep in the tree

Reply:
That doesn't matter. The executable is 4 levels deep.

Reply:
Ah. This is a Visual Studio thing I suppose? I always run this with `dotnet run` in the `NativeFileDialogs.CppSharpGenerator` directory since I use VSCode.

I'd rather still not change this

Reply:
I'll change it back